### ISSUE TITLE
Overhaul branch protection rules

### ISSUE THREAD

**Issue by tsalo** [ME-ICA/tedana#1118]
### Summary

I think we should drop the number of required approvals from 2 to 1 _and_ we should stop dismissing reviews as stale on new commits. I think we can trust contributors to request new reviews if their changes invalidate old reviews.

**Comment by handwerkerd** [comment]
I agree. I could see requiring only 1 review, but adding to the contributors guide that 2 reviews are expected for PRs that include substantive functionality or code changes. We can trust contributors to self-identify which ones those are.

**Comment by tsalo** [comment]
Everyone's good with this. I'll handle the documentation changes and under-the-hood settings.
